nixos/tests/installer.nix

```diff
@@ -68,160 +67,167 @@ let
                   , testCloneConfig
                   }:
     let
+
+      # FIXME taken from tests/boot.nix - should probably be moved to a common location
+      pythonDict = params: "\n    {\n        ${concatStringsSep ",\n        " (mapAttrsToList (name: param: "\"${name}\": \"${param}\"") params)},\n    }\n";
```

Comment:
❌ Just noting here that we need a proper answer to the FIXME before merging this :).

Reply:
(Being handled in another comments series.)